client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/ClinicalDetails.tsx

```diff
@@ -157,21 +158,23 @@ const ClinicalDetails: React.FC = (): JSX.Element => {
                     getOptionLabel={(option) => option.displayName}
                     inputValue={isolationStreetName}
                     onChange={(event, selectedStreet) => {
-                        selectedStreet && updateIsolationAddress(ClinicalDetailsFields.ISOLATION_STREET, selectedStreet.id)}
+                        selectedStreet && updateIsolationAddress(ClinicalDetailsFields.ISOLATION_STREET, selectedStreet.id)
+                    }
                     }
                     onInputChange={(event, selectedStreetName) => {
                         setIsolationStreetName(selectedStreetName);
                     }}
                     renderInput={(params) =>
                         <CircleTextField
+                            test-id='currentQuarantineStreet'
```

Comment:
Same

Reply:
`testId` is a name of a paramter_, while `test-id` is the name of the actual HTML attribute.
Some components need to pass a parameter to sub-components and some don't.